gcp-bundle/LICENSE

```diff
@@ -325,24 +315,30 @@ License: The Apache Software License, Version 2.0 - http://www.apache.org/licens
 
 --------------------------------------------------------------------------------
 
-Group: com.google.code.gson  Name: gson  Version: 2.10.1
+Group: com.google.code.findbugs  Name: jsr305  Version: 3.0.2
+Project URL: http://findbugs.sourceforge.net/
+License: The Apache Software License, Version 2.0 - http://www.apache.org/licenses/LICENSE-2.0.txt
```

Comment:
I think this needs to be excluded.

Reply:
jsr305 mention has been removed from `LICENSE` already.